onnx/version_converter/convert.h

```diff
@@ -476,6 +476,7 @@ class DefaultVersionConverter : public BaseVersionConverter {
     registerAdapter(make_unique<CompatibleAdapter>("Constant", OpSetID(13), OpSetID(12)));
     registerAdapter(make_unique<AxesInputToAttribute>("ReduceSum", OpSetID(13), OpSetID(12)));
     registerAdapter(make_unique<AxesInputToAttribute>("Squeeze", OpSetID(13), OpSetID(12)));
+    registerAdapter(make_unique<CompatibleAdapter>("Sub", OpSetID(13), OpSetID(12)));
```

Comment:
I don't think we can use a CompatibleAdapter. IIUC, version 13 adds support for bfloat16, right? If so, the converter should fail if the input-type is bfloat16, and otherwise it can succeed (with no changes).